projects/crates.io/gitui/package.yml

```diff
@@ -0,0 +1,26 @@
+distributable:
+  url: https://github.com/extrawurst/gitui/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/gitui
+
+versions:
+  github: extrawurst/gitui/tags
+  strip: /v/
+
+build:
+  dependencies:
+    perl.org: '*'
+    openssl.org: '*'
```

Comment:
almost certainly is a runtime dependency. Move to the root `dependencies:` node.